pkg/kubelet/kubelet_pods_test.go

```diff
@@ -1766,6 +1766,7 @@ func waitingStateWithReason(cName, reason string) v1.ContainerStatus {
 		State: v1.ContainerState{
 			Waiting: &v1.ContainerStateWaiting{Reason: reason},
 		},
+		Started: &boolFalse,
```

Comment:
can you please add some tests for boolTrue?

Reply:
this can only be tested if a probe_manager is involved, so not in that file